sunkit_image/tests/test_asda.py

```diff
@@ -14,31 +14,34 @@ def test_asda_artificial():
     vmax = 2.0  # rotating speed
     rmax = 50  # radius
     ratio = 0.2  # ratio of expanding speed over rotating speed
-    with pytest.raises(ValueError, match="Keyword 'r' must be an integer"):
-        lo = asda.Lamb_Oseen(vmax=vmax, rmax=rmax, ratio_vradial=ratio, factor=1, r=1.2)
+    alpha = 1.256430
+    rcore = rmax / np.sqrt(alpha)
+    gamma = 2 * np.pi * vmax * rmax * (1 + 1 / (2 * alpha))
 
-    with pytest.raises(ValueError, match="Keyword 'factor' must be an integer"):
```

Comment:
These were removed, do they exist for the functions?

Reply:
Yes they do I can make tests to cover them